msteams-platform/tabs/how-to/authentication/tab-sso-graph-api.md

```diff
@@ -116,7 +116,7 @@ You can configure authentication for multiple platforms as long as the URL is un
 
 You need to acquire an access token for Microsoft Graph. You can do so by using Microsoft Entra on-behalf-of (OBO) flow.
 
-The current implementation for single sign-on (SSO) is limited to user-level permissions, which aren't usable for making Graph calls. To get the permissions and scopes needed to make a Graph call, SSO apps must implement a custom web service to exchange the token received from the Teams JavaScript library for a token that includes the needed scopes. You can use Microsoft Authentication Library (MSAL) for fetching the token from the client side.
+The current implementation for SSO is limited to user-level permissions, which aren't usable for making Graph calls. To get the permissions and scopes needed to make a Graph call, SSO apps must implement a custom web service to exchange the token received from the Teams JavaScript library for a token that includes the needed scopes. You can use Microsoft Authentication Library (MSAL) for fetching the token from the client side.
```

Comment:
SSO should reamain single sign-on (SSO) to remain consistent with format above and to provide clarity on the acronym.

Reply:
Updated.